widgets/nuxeo-selectivity.html

```diff
@@ -329,11 +329,17 @@
           templates: {
             resultItem: function(options) {
               return (
-                  '<div class="selectivity-result-item' +
-                  (options.disabled ? ' disabled' : '') +
-                  '" data-item-id="' + this.escapeHTML(options.id) + '">' +
-                  this.resultFormatter(options.item)
-                  + '</div>');
+                  '<div class="selectivity-result-item' + (options.disabled ? ' disabled' : '') +
+                  '" style="padding-left:' + (7 + (10 * options.depth)) + 'px" data-item-id="' +
```

Comment:
nit: missing space after `padding-left:`:
```suggestion
                  '" style="padding-left: ' + (7 + (10 * options.depth)) + 'px" data-item-id="' +
```